feature/security/gnsi/credentialz/tests/README.md

```diff
@@ -63,6 +63,19 @@ stream.Send(
 )
 ```
 
+### Configure and enable GLOME
+
+```
+stream.Send(
+    RotateHostParametersRequest {
+        enabled: true,
+        key: "4242424242424242424242424242424242424242424242",
+        key_version: 4,
+        url_prefix: "https://fake-prefix.xyz",
```

Comment:
you might consider a domain that can't exist.
I don't (however) have a guess on that anymore :( .example ? (that does, so far, not exist)

Reply:
Done. TLD .invalid looks good enough to me.